src/coreclr/nativeaot/System.Private.CoreLib/src/System/Diagnostics/Debugger.cs

```diff
@@ -74,5 +74,16 @@ public static bool IsLogging()
             }
             return false;
         }
+
+        /// <summary>
+        /// If a .NET Debugger is attached with break on user-unhandled exception enabled and a method attributed with
+        /// DebuggerDisableUserUnhandledExceptionsAttribute calls this method, the debugger will break with the
+        /// <paramref name="exception"/> details.
+        /// </summary>
+        /// <param name="exception">The user-unhandled exception.</param>
+        [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
+        public static void BreakForUserUnhandledException(Exception exception)
```

Comment:
Rather than have this in three places, can you add a Debugger.cs to the System.Private.Corelib under libraries, add it to the shared proj there, and make all of these files be partial? That way we can have this defined just once... I'm a little surprised we don't already have that, but I guess very little has been shareable until now.